content/en/tracing/setup_overview/proxy_setup/_index.md

```diff
@@ -53,66 +53,66 @@ Datadog APM is included in Envoy v1.9.0 and newer.
 
 ## Enabling Datadog APM
 
-**Note**: The example configuration below is for Envoy v1.14.
-Example configurations for older versions can be found [here][1]
+**Note**: The example configuration below is for Envoy v1.19.
+Example configurations for other versions can be found [in the `dd-opentracing-cpp` GitHub repo][1].
 
-Three settings are required to enable Datadog APM in Envoy:
+The following settings are required to enable Datadog APM in Envoy:
 
 - a cluster for submitting traces to the Datadog Agent
 - `tracing` configuration to enable the Datadog APM extension
```

Comment:
This isn't needed anymore. Tracing is now configured in the `http_connection_manager` section.